src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -292,259 +300,374 @@ export function DistributionForm({
-                    onClick={async () => {
-                      // use the authed api to download the CSV
-                      if (epoch.number) {
-                        const csv = await downloadCSV(epoch.number);
-                        if (csv?.file) {
-                          const a = document.createElement('a');
-                          a.download = `${circle?.organization.name}-${circle?.name}-epoch-${epoch.number}.csv`;
-                          a.href = csv.file;
-                          a.click();
-                          a.href = '';
-                        }
-                      }
-                      return false;
-                    }}
-                  >
-                    Export CSV
-                  </Button>
-                )}
-              </Box>
-            )}
+                </Box>
+              </TwoColumnLayout>
 
-            {submitting && <LoadingModal visible />}
-          </Box>
-        )}
+              {submitting && <LoadingModal visible />}
+            </>
+          )}
+        </Panel>
+        {(fixedDist || circleDist) && <Summary distribution={fixedDist} />}
+        <Flex css={{ justifyContent: 'center', mt: '$xl', mb: '$xl' }}>
+          {!fixedDist ? (
+            fixedPaymentTokenSel.length ? (
+              <Button
+                color="primary"
+                outlined
+                size="large"
+                disabled={submitting || !sufficientFixedPaymentTokens}
+                fullWidth
+              >
+                {sufficientFixedPaymentTokens
+                  ? submitting
+                    ? `Submitting...`
+                    : `Submit ${fixedPaymentTokenSel[0].symbol} Vault Distribution`
+                  : `Insufficient Tokens`}
+              </Button>
+            ) : (
+              <Button
+                type="button"
+                color="primary"
+                outlined
+                size="large"
+                fullWidth
+                onClick={async () => {
+                  // use the authed api to download the CSV
+                  if (epoch.number) {
+                    const csv = await downloadCSV(epoch.number);
+                    if (csv?.file) {
+                      const a = document.createElement('a');
+                      a.download = `${circle?.organization.name}-${circle?.name}-epoch-${epoch.number}.csv`;
+                      a.href = csv.file;
+                      a.click();
+                      a.href = '';
+                    }
+                  }
+                  return false;
+                }}
+              >
+                Export CSV
+              </Button>
+            )
+          ) : (
+            <EtherscanButton distribution={fixedDist} />
+          )}
+        </Flex>
       </form>
-    </Box>
+    </TwoColumnLayout>
   );
 }
+
+const Summary = ({
+  distribution,
+}: {
+  distribution: EpochDataResult['distributions'][0] | undefined;
+}) => {
+  return (
+    <Flex
```

Comment:
if `!distribution`, should we just return null instead of an empty Flex?

Reply:
it would look like this if its a non combined distribution without the Flex box

<img width="1374" alt="image" src="https://user-images.githubusercontent.com/17747090/180659571-d9b717de-9c2c-4cbb-a8e0-ad18a124eaca.png">

Reply:
> @zashton there are still a few more open comments from my review; maybe you missed them because DistributionForm's diff is collapsed by default in Github?

yeah my bad for missing, some of the review got hidden by default. Have added the changes